test/quoteSpec.js

```diff
@@ -344,21 +344,6 @@ describe('Quotes', function () {
         .end()
     })
 
-    it('should return a valid Payment Template object', function * () {
```

Comment:
Can you replace this test with one that validates it against whatever schema it should match?